### ISSUE TITLE
Meta-issue: Show list of shared rooms

### ISSUE THREAD

**Issue by Half-Shot** [vector-im/riot-web#14339]
This issue will be used to track the various PRs I've written to add a shared room list into the user panel.

**Comment by Mikaela** [comment]
Keywords: common groups (and common rooms or rooms in common), shared groups. I found this with my third search attempt.

**Comment by ShadowJonathan** [comment]
Additional keywords: "Mutual servers" from discord

**Comment by ShadowJonathan** [comment]
This came up in HQ: it'd be nice if users could also be able to "filter" DMs to ones from people they have a room in common with, and "delegate" the rest to a "bin" which is only accessable via a menu, sorta like Spam

**Comment by ShadowJonathan** [comment]
@Half-Shot has indicated that, due to work pressure, these issues are open to be picked up by other people, so if anyone wants to take a crack at this, feel free :D

**Comment by ShadowJonathan** [comment]
Current status as of 6 nov; https://github.com/matrix-org/matrix-react-sdk/pull/4897 is blocked on https://github.com/matrix-org/synapse/issues/10339

**Comment by ShadowJonathan** [comment]
https://github.com/vector-im/element-ios/issues/4799 has been closed, but in the meantime, the MSC switched from using `shared_rooms` to `mutual_rooms`, so this has to be changed in the JS SDK before https://github.com/matrix-org/matrix-react-sdk/pull/4897 can be refactored, re-opened or pushed forward.

**Comment by ShadowJonathan** [comment]
This issue is now a duplicate of https://github.com/vector-im/element-meta/issues/371
